docs/source/references/api_spec.md

```diff
@@ -248,6 +248,8 @@ Also returns current status of peer in json string:
 ```
 __CAUTION__: Almost all fields are 64-bit integers and should be handled with care in JavaScript. Only the `nanos` field is 32-bit integer. See `iroha_telemetry::metrics::Status`.
 
+**Sub-routing**: To obtain the value of a specific field, one can append the name of the field to the path, e.g. `status/peers`. This returns the corresponding JSON value, inline, so strings are quoted, numbers are not and maps are presented as above. 
```

Comment:
```suggestion
**Sub-routing**: To obtain the value of a specific field, one can append the name of the field to the path, e.g. `status/peers`. This returns the corresponding JSON value, inlined
```

This maybe sounds redundant?

Reply:
Can't be too careful. This *might* be redundant, but it saves the reader some thinking.